src/TwigComponent/src/Twig/TwigPreLexer.php

```diff
@@ -167,7 +167,7 @@ public function preLexComponents(string $input): string
             throw new SyntaxError(sprintf('Expected closing tag "</twig:%s>" not found.', $lastComponent), $this->line);
         }
 
-        return $output;
+        return $this->processRequires($output);
```

Comment:
What do you think about dispatching an event `PostTwigPrelexing` and putting the processRequires function's content in a Subscriber?